tests/api-diff/run.t

```diff
@@ -29,12 +29,71 @@ there should be no diff:
   $ api-diff ref.cmi ref.cmi
   API unchanged!
 
-Adding a value:
+## Different .cmi files for type tests:
+
+Generate a basic `.mli` files with two types and a function
```

Comment:
I tried using this method to sort of decouple it from the line 14 comment. You see for line 14, it checks for two identical types. I did the markdown syntax on the comments to be able to differentiate type tests of the same file and type tests of two different files. I think if mixed together, the test file could be less readable. So it was a step to enhance readability. @shonfeder

Reply:
Sorry I wasn't clear: my question was just about the line 

```
Generate a basic `.mli` files with two types and a function
```

This is not associated with anything in particular as far as can tell. The markdown-like formatting is fine IMO, and there is precedent for that in this file, but this line seems out of place.

Reply:
This line came with the original `ref.mli` as an explanation of creating a base file that has a two types and a function. Then from the base file we can describe changes to files. For instance:
```
Generate a file with an added type
```
I also see it says `files` instead of `file`. Please let me know if my understanding is correct, I can remove it if needed.

Reply:
That is what I thought. This comment is not relevant in this context tho, as we are not creating the original `ref.mil` here. I suggest just removing this line

```suggestion
```

Reply:
Thanks for the suggestion. I have updated the file, it's ready for review @shonfeder